stores/milvus-documentstore/milvus.py

```diff
@@ -0,0 +1,675 @@
+from typing import Any, Dict, Generator, List, Optional, Union
```

Comment:
I think this file is a lost copy? There's an identical one under `milvus-documentstore/milvus_documentstore`